### ISSUE TITLE
[Request]: Design components for Figma or XD

### ISSUE THREAD

**Issue by linuxexp** [GeekyAnts/NativeBase#3790]
Hi, thank you for native base 3.0, Someone who has used native base since its inception, I am very excited to see how far 3.0 has come. Are there any component design files available for Figma or Adobe XD?

**Comment by Ajamuar** [comment]
Hey @linuxexp, we are currently working on a design file and it should be available soon. We'll keep you posted.

**Comment by shane-mcg** [comment]
Is there any sort of ETA on that? We're looking for something to use for an upcoming project and are considering NativeBase. Having figma components would be a huge help

**Comment by soast** [comment]
Yes. A lot of great work is being done here so I'm rooting for NativeBase use for our next project. It would help immensely to convince the team if UX can work out the screens in Figma (or Sketch/Adobe XD) directly based on NativeBase components.

**Comment by Legym** [comment]
Just wanted to see if there has been any progress on this? Would be really nice to have. Great work so far on the library

**Comment by sanketsahu** [comment]
We are making some significant progress here. We are using Figma with Auto layout, and we are building components, states and everything close to code. We hope to release it by mid of December. Thank you so much!

**Comment by initialvic3** [comment]
Hey just wondering if this is going to be open source or part of the startup+ package? Thanks!

**Comment by tanishabiswas** [comment]
Thank you so much for your patience. The long wait is finally over ! The NativeBase Figma Library is live! Support us on https://www.figma.com/@nativebase. We would love to hear your feedbacks.
